docs/en/observability/tab-widgets/uptime-monitoring/run-test-configure-content.asciidoc

```diff
@@ -85,6 +85,8 @@ Where to deploy the monitor. Monitors can be deployed in multiple locations so t
 Control the monitor's download speeds, upload speeds, and latency to simulate your application's behavior on slower or laggier networks.
 `screenshot` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/common_types.ts#L190[`ScreenshotOptions`])::
 Control whether or not to capture screenshots. Options include `'on'`, `'off'`, or `'only-on-failure'`.
+`privateLocations` (`Array<string>`)::
```

Comment:
I suppose it is confusing here that we only have `locations` above, if we do use the language `public locations` elsewhere in the docs. Maybe we should ultimately change the API to just have `publicLocations` and `privateLocations`. Thoughts anyone?

Reply:
I was wondering that too, maybe `publicLocations` would be better to avoid ambiguity.  We could add this (say in 8.5), leaving `locations` as well (supporting both and combining them if both defined) until a future release when we remove `locations` as that'll be a breaking change (ok in beta for this I think).

Reply:
I've opened https://github.com/elastic/synthetics/issues/582 to track